src/Tizen.NUI/src/public/Common/Color.cs

```diff
@@ -1484,100 +1506,76 @@ private static bool IsInvalidValue(ref float value)
 
         /// This will not be public opened.
         [EditorBrowsable(EditorBrowsableState.Never)]
-        protected override void ReleaseSwigCPtr(System.Runtime.InteropServices.HandleRef swigCPtr)
+        public new void Dispose()
         {
-            Interop.Vector4.DeleteVector4(swigCPtr);
         }
 
-        private Color Add(Color rhs)
+        /// This will not be public opened.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected override void Dispose(bool disposing)
         {
-            Color ret = new Color(Interop.Vector4.Add(SwigCPtr, Color.getCPtr(rhs)), true);
-            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
-            return ret;
+            callback = null;
```

Comment:
Please write some comments why we don't upcall the `base.Dispose(disposing);` here.

Reply:
This patch is using to remove the DALi instance from Color, so it don't need to call base.Dispose(disposing).